File: examples/cmp/cpp/gen.cpp
#include <bits/stdc++.h>
using namespace std;

int main(int argc, char* argv[]) {
    // Generator for Maximum Subarray Problem

    // quicktest passes a seed as an argument: $ ./main seed testcase
    int seed = stoi(string(argv[1]));
    srand(seed);

    const int N = int(1e5);
    const int Ai = int(1e5);

    int n = rand() % N + 1;
    cout << n << "\n";

    default_random_engine generator;
    uniform_int_distribution<int> distribution(-Ai, Ai);

    for(int i = 0; i < n; i++) {
        if(i > 0) cout << " ";
        cout << distribution(generator);
    }
 
    return 0;
}

File: examples/cmp/python/gen.py
import random
import sys

# Generator for Maximum Subarray Problem

# quicktest passes a seed as an argument: $ python3 main.py seed testcase
seed = int(sys.argv[1])
random.seed(seed)

N = int(1e3)
Ai = int(1e2)

n = int(random.uniform(1, N))
print(n)
values = [int(random.uniform(-Ai, Ai)) for _ in range(n)]
print(*values)